ZenPacks/zenoss/ZenPackLib/lib/libexec/ZPLCommand.py

```diff
@@ -28,16 +27,31 @@
 from ..params.ZenPackSpecParams import ZenPackSpecParams
 from ..params.DeviceClassSpecParams import DeviceClassSpecParams
 from ..params.RRDTemplateSpecParams import RRDTemplateSpecParams
+from ..params.EventClassSpecParams import EventClassSpecParams
+from ..params.EventClassMappingSpecParams import EventClassMappingSpecParams
 from ..resources.templates import SETUP_PY
 from ..helpers.WarningLoader import WarningLoader
 from ..helpers.Dumper import Dumper
 from ..helpers.Loader import Loader
 from ..helpers.utils import optimize_yaml
 
+unused(Globals)
+
+
+def stripped_yaml_dump(specparams, Dumper=Dumper):
+    outputfile = yaml.dump(specparams, Dumper=Dumper)
+
+    # tweak the yaml slightly.
+    outputfile = outputfile.replace("__builtin__.object", "object")
+    outputfile = re.sub(r"!!float '(\d+)'", r"\1", outputfile)
```

Comment:
can you confirm that these tags are still output to yaml?  I added a representer for float and the "!ZenPackSpec" should be gone too